demos/continuous_batching/README.md

```diff
@@ -32,7 +33,7 @@ Here, we will also define the LLM engine parameters inside the `graph.pbtxt`.
 Install python dependencies for the conversion script:
 ```bash
 export PIP_EXTRA_INDEX_URL="https://download.pytorch.org/whl/cpu"
-pip3 install "optimum-intel[nncf,openvino]"@git+https://github.com/huggingface/optimum-intel.git@xeon openvino-tokenizers transformers==4.41.2
+pip3 install "optimum-intel[nncf,openvino]"@git+https://github.com/huggingface/optimum-intel.git@main openvino-tokenizers transformers
```

Comment:
maybe it would be the best to have the condition here:
if using public image - install requirements from release branch - set url to requiremetns.txt
if rebuilding the image, you can use the latest version or requirements - required for llama3.1 models

Reply:
I don't think it is a good idea, it would require us to remember to add requirements.txt with frozen dependencies during **every release** we make